lineman/app/models/discussion_model.coffee

```diff
@@ -109,7 +109,7 @@ angular.module('loomioApp').factory 'DiscussionModel', (BaseModel, AppConfig) ->
       @remote.patchMember @keyOrId(), 'set_volume', { volume: volume }
 
     toggleStar: ->
-      @remote.patchMember @keyOrId(), if @starred then 'unstar' else 'star'
```

Comment:
Oh, would you mind tearing out the `toggle` method in the star_toggle_controller as well? I don't think we need it anymore (or the $scope.$on 'starToggled' call in dashboard page controller.. I believe the new angular record store updates have fixed the bug that those were covering)